test/config_test/config_test.cc

```diff
@@ -92,8 +92,13 @@ uint32_t run(const std::string& directory) {
   RELEASE_ASSERT(::chdir(directory.c_str()) == 0);
   uint32_t num_tested = 0;
   for (const std::string& filename : TestUtility::listFiles(directory, false)) {
-    ConfigTest config(filename);
+    Server::TestOptionsImpl options(filename, Network::Address::IpVersion::v6);
+    ConfigTest test1(options);
     num_tested++;
+    if (StringUtil::endsWith(filename, ".json") || StringUtil::endsWith(filename, ".yaml")) {
```

Comment:
I'm not getting a strong feeling for test coverage here, can you elaborate?

Reply:
I got confused after doing the json only version. It now depends on which version of the config we are using. I updated the test and added a comment.